webots-docker/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+python3 docker-config.py ${HOME}/hlvs_webots/controllers/referee/game.json
+rm docker-config.py
```

Comment:
How come you're removing this file after you run it? It's unclear to me. What am I missing?